gadgets/trace_dns/README.mdx

```diff
@@ -19,52 +19,165 @@ Running the gadget:
         ```bash
         $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/trace_dns:latest [flags]
         ```
+        :::tip
+        If you want to try the gadget without installation you can check [kubectl node debug](https://inspektor-gadget.io/docs/latest/quick-start#kubectl-node-debug).
```

Comment:
Should we add those tips in all gadgets?

Reply:
Good point, I moved it to: https://github.com/inspektor-gadget/inspektor-gadget/pull/3433